src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/TouchEffect.shared.cs

```diff
@@ -0,0 +1,1103 @@
+﻿using System.Windows.Input;
+using Xamarin.Forms;
+using System;
+using System.Linq;
+
+namespace Xamarin.CommunityToolkit.Effects
+{
+	public class TouchEffect : RoutingEffect
+	{
+		public event EventHandler<TouchStatusChangedEventArgs> StatusChanged;
```

Comment:
Should we use `WeakeEventManager`? Just wondering if makes sense to make this a default on XCT.
@brminnick thoughts?

Reply:
Anyway, I think it can't be worse if we add it.

Added